index.html

```diff
@@ -279,5 +279,34 @@ <h3 class="titleA">About My self</h3>
                 </li>
               </ul>
             </section>
-  
-  
\ No newline at end of file
+            <!--Works Section contact page-->
+            <section class="page5">
+              <div class="@continer">
+                <div class="bag">
+               
+                <div class="h1">
+                  <form>
+                  <ul><li class="h2"> <h2>Contact me</h2></li>
+                  <li class="p1"> <p>If you have an application you<br>are interested in developing, a <br>feature that you need built or a <br>project that needs coding.<br> I’d love to help with it!</p></li></ul>
+                  <div class="warpare">
+                    <input type="User name" required="required">
+                    <label for="">User name:</label>
+                  </div>
+                      <div class="salar">
+                        <input type="email" required="required">
+                          <label for="">Email:</labe></div>
+                            <div class="sayed">
+                              <input type="User name" required="required">
+                              <label for="">Write your message here</label>
+                            </div>
+                    <div class="btn4">
+                      <button type="submit" class="submit-btn">Get in touch
```

Comment:
- [OPTIONAL] This was not also pointed out by the previous review, You did great adding the button with a `type="submit"`. But as required your button should have the interactive styling like the hover. I'll suggest you look at the Figma designs nd get the background and text color for the hover state of the button and apply them to your CSS. :+1: :+1: